python/Framework.py

```diff
@@ -396,8 +372,7 @@ def smearJets(self, resolutionFile=None, scaleFactorFile=None):
         if self.verbose:
             print("New jets and MET collections: %r and %r" % (self.__miniaod_jet_collection, self.__miniaod_met_collection))
 
-        self.__jer_done = True
-
+    @dep(before=("create", "muonScale"), performs=("correction", "muonScale"))
     def applyMuonCorrection(self, type):
```

Comment:
Same here, muon scale and muon correction are not necessarily the exact same thing

Reply:
Yes... but "correction" can mean pretty much anything, and this is all momentum calibration, therefore I'd call this a "scale correction" (in the hypothetical case where we need to do some ID I think this is clearer), but that's a personal opinion, I don't feel strongly about this.

Reply:
boh ok, for this one we can skip, I agree !